src/Badge/Model/UseCase/CreateDownloadsBadge.php

```diff
@@ -50,7 +50,7 @@ public function createDownloadsBadge(string $repository, string $type, string $f
      */
     protected function prepareText(Package $package, ?string $context = null)
     {
-        $text = $this->normalizer->normalize($package->getPackageDownloads($context));
+        $text = $this->normalizer->normalize($package->getPackageDownloads($context ?? 'total'));
```

Comment:
I would put that as a default value of the argument, and remove the nullability there (if possible)